hs_core/page_processors.py

```diff
@@ -314,8 +315,9 @@ def get_page_context(page, user, resource_edit=False, extended_metadata_layout=N
         temporal_coverage = temporal_coverages[0]
         start_date = parser.parse(temporal_coverage.value['start'])
         end_date = parser.parse(temporal_coverage.value['end'])
-        temporal_coverage_data_dict['start'] = start_date.strftime('%m-%d-%Y')
-        temporal_coverage_data_dict['end'] = end_date.strftime('%m-%d-%Y')
+        dt_template = '{0.year:4d}-{0.month:02d}-{0.day:02d}'
```

Comment:
Is there a reason why this isn't a named template?  I guess if it's a format we don't want to encourage then best to omit as a named template.